specification/attestation/data-plane/Microsoft.Attestation/preview/2018-09-01-preview/attestation.json

```diff
@@ -0,0 +1,340 @@
+                        "description": "Error response describing why the operation failed",
+                        "schema": {
+                            "$ref": "#/definitions/CloudError"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Delete Policy": {         
+                      "$ref": "./examples/Delete_Policy.json"         
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "CloudError": {
+        "x-ms-external": true,
+        "properties": {
+            "error": {
+            "$ref": "#/definitions/CloudErrorBody"
+            }
+        },
+        "description": "An error response from Attestation."
+        },
+        "CloudErrorBody": {
+        "x-ms-external": true,
+        "properties": {
+            "code": {
+            "type": "string",
+            "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+            },
+            "message": {
+            "type": "string",
+            "description": "A message describing the error, intended to be suitable for displaying in a user interface."
+            }
+        },
+        "description": "An error response from Attestation."
+        },
+        "AttestationPolicy": {
+            "type": "object",
+            "properties": {
+            "policy": {
+                "description": "String-encoded attestation policy document.",
+                "type": "string"
+            }
+            }
+        }
+    },
+    "parameters": {
+        "ApiVersionParameter": {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Client API version.",
+            "enum": ["2018-09-01-preview"]
+        },
+        "TeeKind": {
+            "description": "Specifies the trusted execution environment to be used to validate the evidence",
+            "name": "tee",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-parameter-location": "method",
+            "x-ms-enum": {
+                "name": "TeeKind",
+                "modelAsString": true,
+                "values":
+                [
+                    { "value": "SgxEnclave", "description": "Intel Software Guard eXtensions" },
+                    { "value": "OpenEnclave", "description": "OpenEnclave extensions to SGX" },
+                    { "value": "CyResComponent", "description": "IoT Edge validation" },
+                    { "value": "AzureGuest", "description": "Azure Guest Attestation" }
+                ]
+            },
+            "enum": [
+                "SgxEnclave",
+                "OpenEnclave",
+                "CyResComponent",
+                "AzureGuest"
+            ]
+        },
+        "AuthorizationParameter": {
```

Comment:
You do not/should not declare the authorization parameter. The security definition section of the document should be enough.

Reply:
What "security definitions section"? We don't have one and I've not seen a reference for how to define one.

Reply:
Ok, looked up the security definitions section. When that is specified, the client can no longer explicitly specify the bearer token. Is the expectation that clients manually add the bearer token header to the generated REST APIs? That seems like an annoying step backwards in terms of API shape.

Reply:
AuthorizationParameter has been removed